### ISSUE TITLE
transitionCheckOptions config causes build failure

### ISSUE THREAD

**Issue by mscwilson** [gradle-nexus/publish-plugin#137]
The instructions from the README caused my Gradle build to fail.

```
nexusPublishing {
    transitionCheckOptions {
        maxRetries = 100
        delayBetween = Duration.ofSeconds(5)
    }
}
```

The error was: `A problem occurred evaluating root project 'my-project'. > Cannot set the property 'maxRetries' because the backing field is final.`

This was fixed by using `.set` instead.
```
nexusPublishing {
    transitionCheckOptions {
        maxRetries.set(100)
        delayBetween.set(Duration.ofSeconds(5))
    }
}
```

**Comment by jendakol** [comment]
I guess the documentation is ahead of releasing. I ran into exactly the same issue (I wonder why 😂 😂 ) => #136

**Comment by szpak** [comment]
Good point. I merged it without thinking about that aspect :-/. I will think if it is easier to rollback it or release a new version :-).
